tasks/extract_filepaths_for_granule/schemas/input.json

```diff
@@ -0,0 +1,244 @@
+                  "type": "string"
+                },
+                "files": {
+                  "type": "array",
+                  "items": [
+                    {
+                      "type": "object",
+                      "properties": {
+                        "key": {
+                          "type": "string"
+                        },
+                        "bucket": {
+                          "type": "string"
+                        },
+                        "fileName": {
+                          "type": "string"
+                        },
+                        "path": {
+                          "type": "string"
+                        },
+                        "url_path": {
+                          "type": "string"
+                        },
+                        "type": {
+                          "type": "string"
+                        },
+                        "duplicate_found": {
+                          "type": "string"
+                        },
+                        "size": {
+                          "type": "integer"
+                        }
+                      },
+                      "required": [
+                        "key",
+                        "bucket",
+                        "fileName",
+                        "path",
+                        "url_path",
+                        "type",
+                        "duplicate_found",
+                        "size"
+                      ]
+                    },
+                    {
+                      "type": "object",
+                      "properties": {
+                        "key": {
+                          "type": "string"
+                        },
+                        "bucket": {
+                          "type": "string"
+                        },
+                        "fileName": {
+                          "type": "string"
+                        },
+                        "path": {
+                          "type": "string"
+                        },
+                        "url_path": {
+                          "type": "string"
+                        },
+                        "type": {
+                          "type": "string"
+                        },
+                        "duplicate_found": {
+                          "type": "string"
+                        },
+                        "size": {
+                          "type": "integer"
+                        }
+                      },
+                      "required": [
+                        "key",
+                        "bucket",
+                        "fileName",
+                        "path",
+                        "url_path",
+                        "type",
+                        "duplicate_found",
+                        "size"
+                      ]
+                    }
+                  ]
+                }
+              },
+              "required": [
```

Comment:
Not sure all these are required are they? per the code don't we only really need `granuleId` and `files`?

Reply:
yes looks like we only need `granuleId` and `files`